src/hooks/index.ts

```diff
@@ -19,3 +19,4 @@ export * from './useSubjectProjects';
 export * from './useTheme';
 export * from './useUser';
 export * from './useWearables';
+export * from './useHandleAppTileEvents';
```

Comment:
Sorry for the nit but can you alphabetize this?

Reply:
Of course. I'll do it in a follow-up PR.